docs/reST/ref/display.rst

```diff
@@ -514,6 +515,10 @@ required).
     * wayland (Linux/Unix)
     * cocoa (OSX/Mac)
 
+   .. Note:: :func:`toggle_fullscreen` doesn't work on Windows
+             unless the window size is in :func:`pygame.display.list_modes()`.
+             See `issue #2380 <https://github.com/pygame/pygame/issues/2380>`_.
+
    .. ## pygame.display.toggle_fullscreen ##
```

Comment:
That only goes for non-SCALED modes, right?

Reply:
I hadn't tested with SCALED modes before, but you're right. I'll update that.